cubedash/summary/_stores.py

```diff
@@ -374,6 +374,7 @@ def _set_product_extent(self, product: ProductSummary):
             for name in product.derived_products
         ]
         fields = dict(
+            name=product.name,
```

Comment:
I preferred not including this in the update statement, as `name` should never change. Though I'm probably just superstitious.

Reply:
(I also had vague memories of there being a performance downside. But this query isn't run often enough for that to matter...)

Reply:
Instead of getting the ID by name and then updating on ID, we could do the update by name and handle the exception and insert then... saves a query. Then we're updating on name only.